server/src/services/getTransactions.ts

```diff
@@ -1,11 +1,40 @@
+import { Decimal } from 'decimal.js';
+
 import { getFormattedDatetimeUtcFromBlockTimestamp, getRangeFilter } from '../../../shared/helpers/datetime.js';
-import { respondWithServerError } from '../../../shared/helpers/errors.js';
 import { type TxActionRow, type TxActionsFilter } from '../../../shared/types';
+import { respondWithServerError } from '../helpers/errors.js';
+import { PoolsCurrencies } from '../models/PoolsCurrencies.js';
 import { TxActions } from '../models/TxActions.js';
 import { TxTasks } from '../models/TxTasks.js';
 
-// eslint-disable-next-line max-lines-per-function
-export const getTransactions = async (request, response) => {
+/**
+ * TODO: Document what this is doing and why. Also consider moving it to the precision helper file.
+ *
+ * @param {number | string} amount
+ * @param {string} currency
+ * @returns {string}
+ */
+async function convertAmount(amount: number | string, currency: string): Promise<string> {
+  if (currency === 'NEAR' || currency === 'wNEAR')

```

Comment:
I find weird that there is no `{}` since it's used anywhere else

Reply:
I agree. I'll see if I can make that change or if our linter prevented it. Ideally our linter would _require_ {} in my opinion.